modules/auth/cms_authenticaton_email_service.e

```diff
@@ -34,15 +34,34 @@ feature -- Access
 
 feature -- Basic Operations
 
-	send_contact_email (a_to, a_content: READABLE_STRING_8)
-			-- Send successful contact message `a_token' to `a_to'.
+	send_account_evaluation (a_user: CMS_USER; a_application, a_url_activate, a_url_reject: READABLE_STRING_8)
+			-- Send new user register to webmaster to confirm or reject itt.
+		local
+			l_message: STRING
+		do
+			create l_message.make_from_string (parameters.account_evaluation)
+			l_message.replace_substring_all ("$user", a_user.name)
+			if attached a_user.email as l_email then
+				l_message.replace_substring_all ("$email", l_email)
+			else
+				l_message.replace_substring_all ("$email", "unknown email")
+			end
+			l_message.replace_substring_all ("$application", a_application)
```

Comment:
we should really implement a basic var expander in text ...
to optimize this .. and also , we could catch variable without any value 
or value without any variable in the template.